common/src/main/scala/hmda/census/records/CensusRecords.scala

```diff
@@ -95,6 +98,18 @@ object CensusRecords {
         )
     }
 
+  val (indexedTract2022: Map[String, Census], indexedCounty2022: Map[String, Census], indexedSmallCounty2022: Map[String, Census]) =
+    parseCensusFile(censusFileName2022).foldLeft((Map[String, Census](), Map[String, Census](), Map[String, Census]())) {
+      case ((m1, m2, m3), c) =>
+        (
+          m1 + (c.toHmdaTract  -> c),
+          m2 + (c.toHmdaCounty -> c),
+          if (c.smallCounty)
+            m3 + (c.toHmdaCounty -> c)
+          else m3
+        )
+    }
```

Comment:
this can probably be extracted out into a method now, looks to be duplicated for every year.

Reply:
@lchen-2101 Can we make a new ticket for the suggested re-factorization and implement/test it later? Since we're close to the filing season, lets not re-factor at the moment.

Reply:
ticket added: https://github.com/cfpb/hmda-platform/issues/4327